pkg/utils/database/postgres.go

```diff
@@ -236,8 +299,14 @@ func (p Postgres) deleteDatabase(admin AdminCredentials) error {
 
 func (p Postgres) deleteUser(admin AdminCredentials) error {
 	delete := fmt.Sprintf("DROP USER \"%s\";", p.User)
+	dropOwned := fmt.Sprintf("DROP OWNED BY \"%s\";", p.User)
 
 	if p.isUserExist(admin) {
+		logrus.Info("DROPPING OWNED")
+		if err := p.executeQuery("postgres", dropOwned, admin); err != nil {
```

Comment:
doesn't it has to be `executeExec` too? since it's not select and doesn't expect return value.

Reply:
Yep, I'll update

Actually, I think the whole postgres.go package should be executeExec(), because executeQuery() returns only error and the returned value is ignored anyway.

Reply:
according to doc, select is to query and update, insert, delete is exec
https://go.dev/doc/database/execute-transactions

Reply:
I'll update. But as I see the whole `executeQuery` is not working here, because we ignore the actual return statement of Query and just returning error. I've got a feeling that `executeExec` should be used everywhere in the package.